test/javascripts/turbolinks_test.coffee

```diff
@@ -135,11 +144,13 @@ describe 'Turbolinks', ->
           assert(Turbolinks.fullPageNavigate.notCalled, 'Should not perform a full page refresh.')
           done()
 
-    describe 'using data-turbolinks-track="true"', ->
-      startFromFixture = (route) ->
-        fixtureHTML = ROUTES[route][2]
-        document.documentElement.innerHTML = fixtureHTML
+    it 'conflicts do not update the browser history stack', (done) ->
```

Comment:
`Turbolinks head asset tracking with conflicts do not update the browser history stack`

hmmmm

maybe it should not be 

`does not update the browser history stack when a conflict is detected`